.pre-commit-config.yaml

```diff
@@ -54,6 +54,11 @@ repos:
     -   id: pretty-format-json               # Checks that all your JSON files have keys that are sorted and indented.
     -   id: trailing-whitespace              # Trims trailing whitespace.
 
+-   repo: https://github.com/jumanjihouse/pre-commit-hooks
```

Comment:
all else equal, i think we should use the officially-provided hook
https://github.com/markdownlint/markdownlint/blob/master/.pre-commit-hooks.yaml

is there a reason you picked this one?

Reply:
Unfortunately, no. I came across a list of pre-commit hooka here: https://pre-commit.com/hooks.html

I will switch it to the official one.